src/Microsoft.PowerShell.MarkdownRender/LinkInlineRenderer.cs

```diff
@@ -16,14 +16,16 @@ internal class LinkInlineRenderer : VT100ObjectRenderer<LinkInline>
     {
         protected override void Write(VT100Renderer renderer, LinkInline obj)
         {
+            string text = obj?.FirstChild?.ToString();
+
             // Format link as image or link.
             if (obj.IsImage)
```

Comment:
If `obj` is `null`, this will cause an NRE. Would the logic be preserved like this:
```csharp
string text = obj?.FirstChild?.ToString();
if (text == null)
{
    return;
}

// Golden path code...
```
?

Reply:
`obj` can never be null, hence removed `?` after `obj`.

We allow text to be null hence the above logic holds.

Reply:
` if (obj.IsImage) ` - can cause an NRE.

Reply:
@iSazonov  `obj` is guaranteed to be not null. If it is null the renderer is not called.